src/setup-dotnet.ts

```diff
@@ -28,6 +28,11 @@ export async function run() {
     //
     const versions = core.getMultilineInput('dotnet-version');
     const installedDotnetVersions: string[] = [];
+    let architecture = core.getInput('architecture');
+
+    if (!architecture) {
+      architecture = '';
+    }
```

Comment:
I'm not sure that we need this check here. If the architecture input is not supplied function core.getInput() will return an empty string by default.